.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -24,6 +24,7 @@ body:
       label: "Steps to reproduce"
       description: |
         - For build failures: list the exact steps including CMake flags (if any).
+        - If the bug pertains to crashing (or segfault), please include a [stacktrace](https://neovim.io/doc/user/dev_tools.html#dev-tools-backtrace).
```

Comment:
Hmm reviewing it again, put here (steps to reproduce) or move to previous section? e.g. "problem" or "before reporting" which might be *a little* better?